_overviews/scala3-book/methods-intro.md

```diff
@@ -9,7 +9,7 @@ next-page: methods-most
 
 
 In Scala 2, _methods_ can be defined inside classes, traits, objects, case classes, and case objects.
-But it gets better: In Scala 3 they can also be defined _outside_ any of those constructs with a new feature named Toplevel definitions.
+But it gets better: In Scala 3 they can also be defined _outside_ any of those constructs (we say that they are "top-level" definitions since they are not nested in another definition).
```

Comment:
If you want, you can include everything in the parentheses after a semi-colon, such as, “any of those constructs; we say that...”

I’m not sure how to articulate what I’m thinking, but I generally use the semi-colon approach, but it’s also fine either way.

Reply:
Changed for a semi-colon :)